### ISSUE TITLE
Missing progress/error messages

### ISSUE THREAD

**Issue by GutHib** [thunderbird/thunderbird-android#7810]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.801

### Problem you are trying to solve

K-9 badly lacks some basic status messages that will tell you what just happened, what K-9 has been doing and if it has been doing fine or ran into any problems.

- When you push the "archive" button, the email simply vanishes without any indication what just happened.
- When you send mail, the message simply vanishes and you’re back in the inbox. A short message like “Mail sent successfully” would do wonders.
- When you retrieve mail, but don't get any, K-9 won't tell you if there wasn't any mail or if the server timed out or an error was found.

### Suggested solution

The fact, that there's no message upon retrieval is the worst - when you don't get any mail, you always need to retrieve AGAIN because you can never be certain if there wasn't some connection glitch the first time.

**Comment by GutHib** [comment]
This is getting incredibly annoying.

Yesterday, there was an issue with outgoing mail. K9 just wouldn't send it. Wasn't in the mood to tell me why, the messages just sat there in the outbox. Maybe the attachments were too big, but who knows. After half an hour or so, K9 felt like telling me they won't be sent - got a truncated message on my main screen: "SocketTimeoutException: failed to connect to smt..."

Clicked on it to see the full issue, but nope - simply arrived at K9, no error message to be found.

Rest of the evening, when trying to retrieve mail, the little circle would only appear for three seconds, then, K9 would loose interest. Again, no messages.

Second attempt: Circle keeps spinning forever. Again, no clue what's going on.

This morning, still the same thing: K9 attempts to retrieve mail, but doesn't. Retrieving mail on my laptop reveals that there is indeed mail to be retrieved. Minutes later, the mail finally shows up in K9 - no clue how it got there.

Two hours ago, I received a message from K9 that it failed to send mail, even though I deleted the remaining mail from my outbox yesterday. Is it simply telling me what happened yesterday? Or did it snatch one of yesterdays mails, bury it in a hole in the yard and is now attempting to send it again? Nobody knows.

Summary: I don't mind bugs - it's fine, as long as you have a faint idea of what you're facing and / or what to expect. In this case, it's the double combo of non-communication that's killing the deal. #1 K9 isn't communicating. #2 The developers behing K9 aren't communicating. Posted on k9mail.app in early April, no replies. Posted here in early May, only to learn that all K9 issues have been poured into the bulk with Thunderbird.

Reminds me of the core issues K9 had a decade ago - reasons why I eventually moved on. Sad to see that nothing has really changed.

**Comment by kewisch** [comment]
I totally agree, not being able to have the errors handy is very annoying. We're looking to change this by bringing the error messages in the app. #8831 will be the basis for this, and then we'll be making use of it for various types of messages. I'm collecting various use cases for that issue by referencing it, when we do the fine grained planning for that issue we'll review the duplicates to make sure we write down all the use cases we want to cover.

When we have that implemented and you are still experiencing issues like this please do re-open or file a new issue. I appreciate that you're coming back to K-9 and care so much that you're filing issues. I hope we can resolve this as part of 8831 and reduce your frustration. Thank you!
